docs/Documentation/Applications/Plexos/run_plexos.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Running Plexos
+parent: Plexos
+grand_parent: Applications
+---
+
+Please follow the [setup instructions](setup_plexos.md) before running the examples. Example scripts for new users are available within the master branch.
+
+!!! note
+    Sometimes newer modules may be availabe in a `test` directory which is hidden by default from the general user base. This obscured release is done to iron out any bugs that may arise during the installation and use of the module while avoiding breaking users existing jobs and workflows. You can use these test modules by running
+
+    ```bash
+    module use /nopt/nrel/apps/modules/test/modulefiles
+    module avail
+    ```
+
+    This should display all of the test modules available in addition to the defaults. We encourage you to reach out to us at HPC-Help@nrel.gov for access if you would like access to these modules.
+
+## Example Run
+
+We will load the requisite modules for running PLEXOS 9.2R06 for this example. Please see the [module compatibility](setup_plexos.md#Loading-the-Appropriate-Modules) chart for loading the correct modules
+
+```bash
+module load gurobi/10.0.2
+module load plexos/9.200R06
+```
+
+Recall that we can only use the Gurobi solver while running the PLEXOS on the NREL cluster.Now that we have the modules loaded, PLEXOS can be called as follows
+
+```bash
+$PLEXOS/PLEXOS64 -n 5_bus_system_v2.xml -m 2024_yr_15percPV_MT_Gurobi
+```
+
+The command above assumes that we are running the model `2024_yr_15percPV_MT_Gurobi` from file `5_bus_system_v2.xml`. PLEXOS 9.0RX requires validating user-credentials for a local 
+PLEXOS account for each run. Therefore, if we ran the above command in an interactive session, we would need to enter the following username and password
+
+```txt
+username : nrelplexos
```

Comment:
Is it ok to post the username/password publicly, or is that something we give out to specific users/groups?

Reply:
I think it's okay to have this password available in the open as the only thing it enables is for someone to run PLEXOS, assuming that they are already on the network and can ping the license server.